setup.cfg

```diff
@@ -54,7 +54,9 @@ install_requires =
     wandb
     numpy
     torch
-    grelu<1.0.7
+    pyfaidx
+    genomepy
+    grelu
```

Comment:
I believe we will have to create a new release of gReLU and set that as the minimum version. Otherwise, the new arguments in [src/decima/model/decima_model.py] will throw an error.

Reply:
`grelu>=1.0.9` added